### PR TITLE
Setup initial styles

### PR DESCRIPTION
**Pull request by ALapina**
## What does this Pull Request do?

Setup initial styles

## Related Issues

- Issue #2  

## Design decision

- styled components
- css color variables (themes) https://styled-components.com/docs/advanced#theming
- font family and weight (Raleway)
- global styles https://scalablecss.com/styled-components-global-styles/
- styled-normalize https://github.com/sergeysova/styled-normalize

## Steps to manually check the changes

start the project - there is background color and etc

## Author's checklist

- [ ] My changes are covered by tests
- [ ] My changes are documented, existing documentation is updated
- [x] My changes follow our code quality guidelines

## Review checklist

- [x] Pull request title contains the correct ticket number
- [x] Pull request is opened against the correct base branch
- [x] Pull request description matches the changes in the code
- [x] There is no apparent way to improve the performance & design of the new code

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+function App() {
+  return (
+    <React.Fragment>
```
Comment (aalekseev) [inline src/components/App.js:5]: qq, why you are using `Fragment` here, if there is only one element that this component returns?

File: src/setupTests.js [hunk 2]
```diff
@@ -1,5 +0,0 @@
-// jest-dom adds custom jest matchers for asserting on DOM nodes.
```
Comment (aalekseev) [inline src/setupTests.js:1]: Would recommend to keep it, since there will be tests, and this file helps to setup them

**Review by aalekseev — COMMENTED** [review]
Finished the review, @ALapina please look over the comments that I left

File: src/components/App.js (hunk 1 above)
Comment (ALapina) [reply src/components/App.js:5]: fix it in https://github.com/ALapina/Weather-app-Challenge/pull/22

File: src/setupTests.js (hunk 2 above)
Comment (ALapina) [reply src/setupTests.js:1]: fix it in https://github.com/ALapina/Weather-app-Challenge/pull/22

**Pull request merged**